.github/workflows/publish.yml

```diff
@@ -72,7 +72,9 @@ jobs:
   releases:
     needs: [linter-master, linter-pull-request, tests]
     name: GoReleaser Build on All OS but Windows
-    if: startsWith(github.ref, 'refs/tags/')
+    # related to https://github.com/actions/runner/issues/2205
+    # since a success() is added by default and skipped jobs make success to fail we need this workaround
```

Comment:
suggestion: you explain what the workaround is, but not what the problem is. Hence it's kind of complicated to understand the problem in the first place. Could you explain it?

Reply:
praise: thanks, nice explanation!